src/site/content/integration.adoc

```diff
@@ -103,6 +90,8 @@ from https://github.com/silb[@silb].
 +
 Apache Shiro support for the Jersey JAX-RS implementation.
 
+NOTE: This module was move into Apache Shiro.
```

Comment:
```suggestion
NOTE: This module was moved into Apache Shiro.
```

or I think "had been moved" might be even better, because it is an action which was completed in the past.